provision/acc_provision/acc_provision.py

```diff
@@ -388,10 +395,12 @@ def config_adjust(args, config, prov_apic, no_random):
             l3out_name = "calico-l3out-fsvi-vlan-%s" % vlan_id
             config["aci_config"]["cluster_l3out"]["name"] = l3out_name
         l3out_name = config["aci_config"]["cluster_l3out"]["name"]
+        config["aci_config"]["cluster_l3out"]["name"] = l3out_name + "-" + get_md5_hash(l3out_name)[:6]
```

Comment:
@snaiksat confirm plz : 
whether cluster_l3out name is passed through input file or not, always append a hash to it.